hybrid/hello_ddc.c

```diff
@@ -14,12 +14,14 @@
 
 int main()
 {
-	pp_cap(read_ddc());
 	uint16_t *some_int_ptr = (uint16_t *) malloc(sizeof(uint16_t));
 	*some_int_ptr = 200;
-	write_ddc((void *__capability) some_int_ptr);
 	uint16_t *__capability ddc_val = (uint16_t *__capability) cheri_ddc_get();
-	pp_cap(cheri_ddc_get());
-	printf("Value in DDC: %d\n", *ddc_val);
+	pp_cap(ddc_val);
+	assert(cheri_address_get(ddc_val) != some_int_ptr);
+	write_ddc((void *__capability) some_int_ptr);
+	ddc_val = (uint16_t *__capability) cheri_ddc_get();
+	assert(cheri_address_get(ddc_val) == some_int_ptr);
+	pp_cap(ddc_val);
```

Comment:
Do we need the `pp_cap`s? I think we can kill them?

Reply:
Sure, we'll do.

Reply:
Fix in bcc4c6d.

Reply:
Addressed in 108c268.